src/ldc/attributes.d

```diff
@@ -223,3 +223,23 @@ immutable weak = _weak();
 private struct _weak
 {
 }
+
+///Readability aliases for compute
```

Comment:
> So It is very unlikely to be accidentally broken.

The dlang spec is not to be trusted. Because DMD's code is so opaque, things break quite often and anything not tested cannot be relied on. It's good you verified there is a test for module UDAs.